monasca_persister/conf/influxdb.py

```diff
@@ -20,6 +20,12 @@
     cfg.StrOpt('database_name',
                help='database name where metrics are stored',
                default='mon'),
+    cfg.BoolOpt('db_per_tenant',
+                help='Whether to use a separate database per tenant',
+                default=False),
+    cfg.BoolOpt('default_retention_policy',
```

Comment:
I added some questions on this upstream - please see those.

Reply:
Addressed